package.json

```diff
@@ -6,7 +6,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "emulators:start": "cd ./firebase && firebase emulators:start --import=./seeds",
+    "emulators:start:clean": "cd ./firebase && firebase emulators:start"
```

Comment:
Can we make this more like a command we see? Something like `npm run ems --clean` or `npm run ems -C`?

Reply:
this is a pretty standard way of script naming. you can't add flags to script names as npm will read them as actual scripts rather than the name of the script